### PR TITLE
Update Connectors Overview page

### PR DESCRIPTION
**Pull request by ShammiL**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Comment by ShammiL** [comment]
The connector store location has been changed to the [current location](https://store-v2.wso2.com/?page=1&product=MI+Connector) for now. This needs to be updated once the connectors are moved to the final location.

File: en/docs/reference/connectors/connectors-overview.md [hunk 1]
```diff
@@ -41,65 +41,65 @@ Using connectors provide the following advantages:
...
+        <td>Connectors act as a bridge between the main integration logic and the third-party application. Even if the application needs to be updated to support new features, the main integration logic of the API version does not need to be changed.</td>
     </tr>
 </table>
 
 ## How to use connectors
 
-When configuring the integration logic, you need to use WSO2 Integration Studio. When ready, you can export the project along with dependency connectors to the integration runtime. See [documentation on adding connectors]({{base_path}}/develop/creating-artifacts/adding-connectors/) for more information.
+When configuring the integration logic, you need to use Micro Integrator for VS Code. When ready, you can export the project along with dependency connectors to the integration runtime. See [documentation on adding connectors]({{base_path}}/develop/creating-artifacts/adding-connectors/) for more information.
 
 See the following video for a quick look at how to use connectors.
```
Comment (chanikag) [inline en/docs/reference/connectors/connectors-overview.md:72]: Let's remove the video, create an issue to track it, until we come up with a proper video.

Comment (ShammiL) [reply en/docs/reference/connectors/connectors-overview.md]: Updated

**Comment by ShammiL** [comment]
The YouTube tutorial needs to be updated. Tracked here: https://github.com/wso2/docs-mi/issues/700

**Review by chanikag — APPROVED** [review]

**Pull request merged**